src/parser/__snapshots__/parse.test.ts.snap

```diff
@@ -109,6 +114,11 @@ exports[`parse can parse strings 5`] = `
     "tagName": "bar",
     "text": "{bar}",
   },
+  TextNode {
+    "kind": "text",
+    "parent": null,
+    "text": " ",
```

Comment:
duplicate of line 24(?)

Reply:
That's because the source string is `foo <foo>bar</foo> {bar}` and there was a bug before where we had `trim` on the text and skipped them if they were empty.